frontstage/templates/sign-in/sign-in.account-not-verified.html

```diff
@@ -8,7 +8,7 @@ <h1 class="ons-u-fs-xl">Verify your email</h1>
 
     <p>When you created your account we sent you an email to verify your email address.</p>
     <p>Please follow the link the email to confirm your email address and sign in.</p>
-    <br>
+    <br />
```

Comment:
Need to remove the extra space (for this and all the others that have random extra space before the /).  Should be <br/>